docs/imviz/import_data.rst

```diff
@@ -131,12 +131,12 @@ Importing regions via the API
 
 If you have a region file supported by :ref:`regions:regions_io`, you
 can load the regions into Imviz as follows. Any unsupported region will
-be skipped with warning and a dictionary of regions that failed to load
+be skipped with a warning and a list of ``(region, reason)`` that failed to load
 will be returned, if any::
 
-    bad_regions = imviz.load_static_regions_from_file("/path/to/data/myregions.reg")
+    bad_regions = imviz.load_regions_from_file("/path/to/data/myregions.reg")
```

Comment:
I can't suggest this properly since the region covers some deleted lines, but I propose putting the code block directly after the "as follows" and breaking the last sentence into two to decrease complexity:

```
If you have a region file supported by :ref:`regions:regions_io`, you
can load the regions into Imviz as follows:

    bad_regions = imviz.load_regions_from_file("/path/to/data/myregions.reg")

Unsupported regions will be skipped and trigger a warning. Those that
failed to load, if any, will be returned as a list of tuples of the
form ``(region, reason)``.
```

Reply:
Thanks. I applied your suggestion and gave you co-author credit.